pymc3/variational/advi.py

```diff
@@ -410,3 +410,8 @@ def rvs(x):
         trace.record(point)
 
     return MultiTrace([trace])
+
+
+def infmean(input_array):
+    """Return the median of the finite values of the array"""
```

Comment:
Good catch, my mistake.
(Meant to skip ci on the fix, sadly looks like I can't cancel it after the fact.)